App/public/css/groupPage.css

```diff
@@ -166,5 +166,23 @@ a{
 .update_expense_button{
     width: 0.6em;
     align-self: center;
-    justify-self: flex-end;
+    color: var(--purple);
+}
+
+
+.delete_expense_button{
+    width: 0.6em;
```

Comment:
I suggest using REM or px for buttons. 

I suggest you to read this: https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/

Reply:
done